Sources/CollectionViewDriver.swift

```diff
@@ -38,14 +38,14 @@ public final class CollectionViewDriver: NSObject {
     /// A flow layout delegate object to receive forwarded events.
     public weak var flowLayoutDelegate: UICollectionViewDelegateFlowLayout?
 
-    private let _emptyViewProvider: EmptyViewProvider?
+    let _emptyViewProvider: EmptyViewProvider?
 
     private var _currentEmptyView: UIView?
 
     // Avoiding a strong reference to prevent a possible retain cycle.
     // This is typically the view controller that owns `self` (the driver).
     // The caller is responsible for retaining this object for the lifetime of the driver.
-    private weak var _cellEventCoordinator: CellEventCoordinator?
+    private(set) weak var _cellEventCoordinator: CellEventCoordinator?
```

Comment:
Could we keep these 2 properties private and pass them to `driverDebugDescription()` ?

Reply:
Done.